Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -1037,7 +1038,7 @@ void MainWindow::BootWiiSystemMenu()
 void MainWindow::NetPlayInit()
 {
   m_netplay_setup_dialog = new NetPlaySetupDialog(this);
-  m_netplay_dialog = new NetPlayDialog(this);
+  m_netplay_dialog = new NetPlayDialog(nullptr);
```

Comment:
It's a common pattern/thing in Qt, to define default parameters for window types' constructors that allow no parenting, so it's likely preferable to simply change:

```cpp
explicit NetPlayDialog(QWidget* parent);
```

to

```cpp
explicit NetPlayDialog(QWidget* parent = nullptr);
```

(this is also what Qt Creator will auto generate for new created classes)